admin.go

```diff
@@ -10,24 +10,36 @@ import (
 	"github.com/grafana/grafana/pkg/api/dtos"
 )
 
-func (c *Client) CreateUserForm(settings dtos.AdminCreateUserForm) error {
+func (c *Client) CreateUserForm(settings dtos.AdminCreateUserForm) (int64, error) {
+	id := int64(0)
 	data, err := json.Marshal(settings)
 	req, err := c.newRequest("POST", "/api/admin/users", bytes.NewBuffer(data))
 	if err != nil {
-		return err
+		return id, err
 	}
 	resp, err := c.Do(req)
 	if err != nil {
-		return err
+		return id, err
+	}
+	if resp.StatusCode != 200 {
+		return id, errors.New(resp.Status)
 	}
 	data, err = ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return err
+		return id, err
 	}
-	if resp.StatusCode != 200 {
-		return errors.New(resp.Status)
+	user := struct {
+		Id int64 `json:"id"`
+	}{}
+	err = json.Unmarshal(data, &user)
+	if err != nil {
+		return id, err
 	}
-	return err
+	return user.Id, err
+}
+
+func (c *Client) CreateUser(email, login, name, password string) (int64, error) {
```

Comment:
Why does this exist in addition to `CreateUserForm` ?

Reply:
I couldn't find a way for the library and provider to use the same `dtos` package since the provider was using a vendor'ed one, so this was a workaround. I guess the package could just export one generic method without the use of the `dtos` package?

Reply:
Hmm, I'm not sure if pulling packages from something that is a client of a library is a good direction. Usually it's the other way around.

I do not think passing in the DTO from a client is necessary (or a good approach here).

Reply:
Ah, sorry. The dtos come from grafana, not the provider. That's OK, then.  :)

Reply:
I think that if there are data structs for all the grafana resources, we should import/use them in the client.

Reply:
That `dtos` package created a compile issue due to it being vendored/not vendored between the two. Maybe we can just have this method use the `User` struct that the other user methods use?